.github/containerscan/allowedlist.yaml

```diff
@@ -1,3 +1,4 @@
 general:
   vulnerabilities:
     - CVE-2019-1010022 # "Will not fix", see https://access.redhat.com/security/cve/CVE-2019-1010022
+    - CVE-2023-2253
```

Comment:
Is this something that can be fixed?
IMAGE: dellemc/dell-replication-init:v1.0.0
FIXES AVAILABLE
[HIGH] CVE-2023-2253 can be fixed by upgrading "github.com/docker/distribution" from "v2.8.1+incompatible" to "2.8.2-beta.1

Reply:
Yes, that's right. _init_ image just includes script and kubectl binary. CVEs that are reported, are from kubectl binary. We update to latest minor version of required kubectl version when available.